isotovideo

```diff
@@ -245,6 +246,7 @@ $bmwqemu::vars{TEST_GIT_HASH} = checkout_git_refspec($bmwqemu::vars{CASEDIR} =>
 # set a default distribution if the tests don't have one
 $testapi::distri = distribution->new;
 
+checkout_wheels($bmwqemu::vars{PRODUCTDIR});
```

Comment:
why productdir? In case of os-autoinst-distri-opensuse that would be something like "products/opensuse/" or "products/sle", shouldn't that rather be "CASEDIR"? Or do you explicitly want to allow loading product-specific plugins? Also how about defining a variable that defaults to PRODUCTDIR if not set?:

```suggestion
checkout_wheels($bmwqemu::vars{WHEELS_DIR} // $bmwqemu::vars{PRODUCTDIR});
```

Reply:
I thought it might be intuitive since PRODUCTDIR defaults to CASEDIR. And having a product-specific wheel configuration will eventually be useful.
WHEELS_DIR is a good idea actually since defaults won't work in all cases anyway.